tests/integration/team_test.py

```diff
@@ -17,34 +17,106 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
+import os
+import re
+
 import pytest
 
 import libnmstate
-from libnmstate.error import NmstateNotImplementedError
+from libnmstate.error import NmstateLibnmError
 from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import Team
 
+from .testlib import assertlib
+from .testlib import cmd as libcmd
+
+
+RPM_NM_TEAM_PLUGIN_PATH = ("rpm", "-ql", "NetworkManager-team")
+
+SYSTEMCTL_RESTART_NM_CMD = ("systemctl", "restart", "NetworkManager")
 
 TEAM0 = "team0"
 
-TEAM0_STATE = {
-    Interface.KEY: [
-        {
-            Interface.NAME: TEAM0,
-            Interface.TYPE: InterfaceType.TEAM,
-            Team.CONFIG_SUBTREE: {
-                Team.PORT_SUBTREE: [{Team.Port.NAME: "eth1"}]
+
+@pytest.mark.skipif(
+    os.environ.get("CI") == "true",
+    reason="Team kmod not available in Travis CI",
+)
+def test_create_team_iface():
+    with team_interface(TEAM0) as team_state:
+        assertlib.assert_state(team_state)
+
+
+@pytest.mark.skipif(
+    os.environ.get("CI") == "true",
+    reason="Team kmod not available in Travis CI",
+)
+def test_edit_team_iface():
+    with team_interface(TEAM0) as team_state:
+        team_state[Team.CONFIG_SUBTREE] = {
+            Team.RUNNER_SUBTREE: {
+                Team.Runner.NAME: Team.Runner.RunnerMode.LOAD_BALANCE
             },
+            Team.PORT_SUBTREE: [{Team.Port.NAME: "eth1"}],
         }
-    ]
-}
+        libnmstate.apply(team_state)
```

Comment:
What about asserting that the new config is in?

Reply:
I fixed a typo and also added the asserting. Thanks :)